babel-watch.js

```diff
@@ -244,6 +245,11 @@ function restartApp() {
     }
   }
 
+  // If debugger mode is present 
+  if(program.debug) {
```

Comment:
Given the prevailing style in this file, I think you need a space after this `if`.